static.go

```diff
@@ -123,7 +123,7 @@ func staticHandler(ctx *Context, log *log.Logger, opt StaticOptions) bool {
 		return false
 	}
 
-	file := ctx.Req.URL.Path
+	file := path.Clean(ctx.Req.URL.Path)
```

Comment:
Looks like the call of `path.Clean` here is ineffective and needless, so we can remove it.